src/components/Medicine/MedicationAdministration/MedicineAdminForm.tsx

```diff
@@ -64,21 +64,26 @@ export const MedicineAdminForm: React.FC<MedicineAdminFormProps> = ({
   const [endTimeError, setEndTimeError] = useState("");
 
   const validateDateTime = (date: Date, isStartTime: boolean): string => {
-    const now = new Date();
-    const authoredOn = new Date(medication.authored_on);
-    const startTime = new Date(administrationRequest.occurrence_period_start);
-
-    if (date > now) {
+    const now = startOfMinute(new Date());
+    const authoredOn = startOfMinute(new Date(medication.authored_on));
+    const startTime = startOfMinute(
+      new Date(administrationRequest.occurrence_period_start),
+    );
+    const roundedDate = startOfMinute(date);
```

Comment:
you could just re-assign the value and skip many other changes below right? 🤔 

```suggestion
    date = startOfMinute(date);
```